Source/FieldSolver/SpectralSolver/SpectralFieldData.H

```diff
@@ -26,6 +26,7 @@ class SpectralFieldData
     // the local MPI rank)
 #ifdef AMREX_USE_GPU
     // Add cuFFT-specific code
```

Comment:
Could you remove the comment `// Add cuFFT-specific code`?

Reply:
Done.